client/src/components/auth/Login.js

```diff
@@ -26,13 +31,13 @@ const Login = () => {
 
   const handlePasswordChange = e => {
     const { value } = e.target;
-    setPassword(value );
+    setPassword(value);
   };
 
   return (
-    <Container>
-      <Typography variant="h2">Login</Typography>
+    <Container style={{ maxWidth: 480, margin: "0 auto", textAlign: "center" }}>
```

Comment:
look at these awesome inline styles, whoever put that in must be a genius ;)

Reply:
I did...I moved them from the Form to the container. SO thank you